src/Admin/BlockAdmin.php

```diff
@@ -206,7 +231,7 @@ public function setContainerBlockTypes(array $containerBlockTypes): void
     }
 
     /**
-     * @param $defaultContainerType
+     * @param array $defaultContainerType
      */
     public function setDefaultContainerType($defaultContainerType): void
```

Comment:
Can't you put array typehint here?

Reply:
Fixed

Reply:
If the master is `7.1` why not use type hints instead of phpdoc as I see there is `:void` already?